packages/syntax/src/languages.ts

```diff
@@ -0,0 +1,76 @@
+import javascript from 'highlight.js/lib/languages/javascript';
```

Comment:
Would love to do this. That said, we'd need to spend some time thinking about how that would work. Right now, the only way we have to prevent importing languages that we don't support is through our build process, which is a heavy lift for applications. The library itself doesn't make this easy.

Reply:
Agreed!